devel/flatbuffers/Portfile

```
@@ -55,6 +52,8 @@ set fb_langs            {
                             ts
                         }
 
+configure.env           FLATBUFFERS_OSX_BUILD_UNIVERSAL=OFF
```

I think this needs to be a configuration option passed to `cmake`, rather than an environment variable.  Maybe try:

```suggestion
configure.args-append   -DFLATBUFFERS_OSX_BUILD_UNIVERSAL=OFF
```